docs/how-to/uc-boot-parameters.rst

```diff
@@ -0,0 +1,89 @@
+How to configure boot parameters on Ubuntu Core 
+================================================
+
+.. note::
+
+    This guide assumes access to features available exclusively to `dedicated Snap Store`_ users.
+
+Ubuntu Core doesn't have the :code:`/etc/default/grub` file mapped for the kernel parameters, nor the `update-grub`_ utility.
+On Ubuntu Core, the kernel parameters can be modified statically or dynamically.
+Static configurations are added during the image build.
+On the other hand, dynamic configurations are set `using snap options`_ on a running system.
+This guide focuses on the dynamic configuration method, suitable for iteratively tuning the operating system.
+
+.. hint::
+    
+    Refer to :doc:`../how-to/uc-image-creation` if you want to create a real-time Ubuntu Core image.
+
+Open a terminal and access your real-time Ubuntu Core instance.
+The instance should be on a physical device rather than a virtual machine.
+While a virtual machine can be used to follow the steps in this guide, it will not provide the benefits of a real-time kernel.
+
+There are two snap options for setting the kernel parameters:
+
+- `system.kernel.cmdline-append`_
+- `system.kernel.dangerous-cmdline-append`_
+
+The first one is used to append permitted kernel boot parameters that are verified against an `allow list`_ in the `gadget snap`_. 
+The second one is used to append any other kernel boot parameters, considered as not analyzed by the gadget snap.
+
+When using the reference gadget snaps (`pc-gadget`_ and `pi-gadget`_), you need to use the `system.kernel.dangerous-cmdline-append` option, as the allow list is not present in them.
+
+When creating a custom `gadget snap`_ for a real-time Ubuntu Core system, it is advisable to add an allow list with the relevant parameters.
+
+The kernel parameters in the snap configurations exactly match the `kernel parameters`_ that would be passed to the kernel on the grub command line, the :code:`GRUB_CMDLINE_LINUX_DEFAULT` on :code:`/etc/default/grub` file. 
+Parameters are passed as a single string in the :code:`key=value` format, with each parameter separated by a space.
+
+In the case of :doc:`real-time kernel parameters <../reference/kernel-boot-parameters>`, the most interesting parameters are `nohz`_, `nohz_full`_ and `irqaffinity`_.
+The :code:`nohz` is used to enable/disable dynamic ticks (possible values are `on` or `off`). 
```

Comment:
This is rather a nitpick also.
But I would suggest some edits for readability similar to this:
https://documentation.ubuntu.com/real-time/en/latest/how-to/tune-irq-affinity/#how-to-tune-irq-affinity

```
...the most interesting parameters are `nohz`_, `nohz_full`_ and `irqaffinity`_.
- :code:`nohz` - Used to enable...
- :code:`nohz_full` - Receives...
```